### DISCUSSION TITLE
Upcoming Talks and Presentations

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#3584] [General]
I think I heard that we had four talks coming up in the near future. I just saw that @aljazerzen has one tomorrow (2 Oct 2023) at QCon...

[PRQL: A Simple, Powerful, Pipelined SQL Replacement](https://qconsf.com/presentation/oct2023/prql-simple-powerful-pipelined-sql-replacement)

**Comment by snth** [comment]
Hi @richb-hanover ,

Here are the links to all four talks:

- [QCon SF, October 2nd, San Francisco, USA: ](https://qconsf.com/presentation/oct2023/prql-simple-powerful-pipelined-sql-replacement)
- [PyconZA, October 5th, Durban, South Africa: ](https://za.pycon.org/talks/31-prql-a-modern-language-for-data-transformation/)
- [Community over Code (ApacheCon), October 9th, Halifax, Canada:  ](https://communityovercode.org/schedule-list/#FT005)
- [data2day, October 12th, Karlsruhe, Germany: ](https://www.data2day.de/veranstaltung-21353-0-prql-a-modern-language-for-data-transformation.html)

We also have a promotional trailer that's on YouTube (https://www.youtube.com/watch?v=cFMe0QZMc4s) and Twitter (https://twitter.com/prql_lang/status/1707507958047723590).

**Reply by vanillajonathan** [reply]
Are recording or streams of any of the talks available?

**Reply by max-sixty** [reply]
QCon unfortunately doesn't stream. Videos will be available, but behind a very tall paywall. Not sure if we can somehow get a version we can post, might be unlikely

The others I hope will be more available; @snth will know more (but is traveling atm)

**Reply by vanillajonathan** [reply]
It would be great if they could be posted on the [@prql-lang](https://www.youtube.com/@prql-lang) YouTube channel.

**Reply by vanillajonathan** [reply]
If you have any slides from the talks maybe you could put them on [SlideShare](https://www.slideshare.net/)?

**Reply by aljazerzen** [reply]
https://github.com/aljazerzen/prql-qcon-2023/blob/main/main.pdf
